modules/telco-ran-sysctls.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/telco-ran-du-rds.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="telco-ran-sysctls_{context}"]
+= Kubelet Settings
+
+Some CNF workloads make use of sysctls which are not in the list of system-wide safe sysctls.
+Generally, network sysctls are namespaced and you can enable them using the `kubeletconfig.experimental` annotation in the `PerformanceProfile` Custom Resource (CR) as a string of JSON in the following form: 
+
+.Example snippet showing allowedUnsafeSysctls
+[source,yaml]
+----
+apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: {{ .metadata.name }}
+  annotations:kubeletconfig.experimental: |
+      {"allowedUnsafeSysctls":["net.ipv6.conf.all.accept_ra"]}
+# ...
+----
+
+[NOTE]
+====
+Although these sysctls are namespaced, they may allow a pod to consume memory or other resources beyond any limits specified in the pod description.
+You must ensure that these sysctls do not exhaust platform resources.
+====
```

Comment:
Can we add a reference to the primary discussion on sysctl use for additional considerations/guidance on their use:
https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/nodes/working-with-containers#nodes-containers-sysctls